src/stacks-hierarchy/Root_ChooseTicketReceiverScreen/Root_ChooseTicketReceiverScreen.tsx

```diff
@@ -23,6 +24,14 @@ export const Root_ChooseTicketReceiverScreen: React.FC<Props> = ({
 
   const focusRef = useFocusOnLoad();
 
+  const handleValidatePhoneNumber = async (phoneNumber: string) => {
+    const result = await getCustomerAccountId(phoneNumber);
+    if (result) {
+      return typeof result === 'string' ? result : 'no_associated_account';
+    }
+    return 'unknown_error';
+  };
```

Comment:
Handle this in the api-method? So not every place in the code where we use the api method needs to handle it.